### PR TITLE
feat(analytics): Add Google Analytics

### PR DESCRIPTION
**Pull request by BYK**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -13,29 +13,45 @@ const queries = require("./src/utils/algolia");
 const getPlugins = () => {
   const plugins = [
     {
-      resolve: "@sentry/gatsby",
+      resolve: '@sentry/gatsby',
```
Comment (dcramer) [inline gatsby-config.js:16]: did the prettier config (thats present in the repo) do this? i worry that if i open it its going to change it all back

File: gatsby-config.js [hunk 2]
```diff
@@ -13,29 +13,45 @@ const queries = require("./src/utils/algolia");
...
+    'gatsby-plugin-zeit-now',
     {
-      resolve: "gatsby-plugin-mdx",
+      resolve: 'gatsby-plugin-google-gtag',
       options: {
-        remarkPlugins: [require("remark-deflist")],
+        // You can add multiple tracking ids and a pageview event will be fired for all of them.
+        trackingIds: [
+          'UA-30327640-1', // Sentry
+          'UA-30327640-3', // www
```
Comment (dcramer) [inline gatsby-config.js:33]: do we actually want www on here? not really sure

Comment (BYK) [reply gatsby-config.js:33]: This was what was done earlier in the docs repo. I'm not a fan of duplication so I'll remove it.

File: gatsby-config.js (hunk 1 above)
Comment (BYK) [reply gatsby-config.js:16]: I run Prettier automatically but I'll make sure the config from the repo was used for this.

Comment (BYK) [reply gatsby-config.js:16]: Just ran `yarn format`, same result. It even changes 14 more files.

Comment (dcramer) [reply gatsby-config.js:16]: hrm maybe we have a fucked up config in develop repo :s

this is what we have in docs

```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": true,
  "singleQuote": false,
  "overrides": [
    {
      "files": "*.mdx",
      "options": {
        "parser": "markdown"
      }
    }
  ]
}
```

**Review by dcramer — APPROVED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (dcramer) [reply gatsby-config.js:16]: is it possible your prettier has some global config? like singleQuotes arent default are they?

i see some of the other formatting issues tho

Comment (BYK) [reply gatsby-config.js:16]: Submitted #151. Will rebase onto that once merged.

**Pull request merged**